lib/jekyll_auth/config.rb

```diff
@@ -3,21 +3,36 @@ def self.config_file
     File.join(Dir.pwd, '_config.yml')
   end
 
+  def self.jekyll_config
+    @config ||= begin
+      jekyll_config = YAML.safe_load_file(config_file)
+    rescue
+      {}
+    end
+  end
```

Comment:
Actually, on second thought, I'm not sure that we should rescue. If the Config has an error, we should fail loudly.